### PR TITLE
Add spell check workflow

### PR DESCRIPTION
**Pull request by hezhizhen**

### CODE REVIEW

File: typos.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
+[default.extend-words]
+ba = "ba"
+fo = "fo"
+enew = "enew"
+tabe = "tabe"
+[files]
+extend-exclude = ["CHANGELOG.md"]
```
Comment (junegunn) [inline typos.toml:8]: Thanks, can you explain why you added this line?

Comment (hezhizhen) [reply typos.toml:8]: Ignore this file.
I think it's better to keep it the same as PRs' titles, even if there're some typos.

Comment (junegunn) [reply typos.toml:8]: The file does not include the titles of the pull requests. I edit the file manually and summarize the important changes.

Comment (hezhizhen) [reply typos.toml:8]: I see. I thought it was automatically generated. Let me include it in the scope of checking.

Comment (hezhizhen) [reply typos.toml:8]: Do you prefer a commit in a PR or multiple commits?

**Review by junegunn — APPROVED** [review]

**Comment by junegunn** [comment]
Merged, thanks!

**Pull request merged**